Compiler/AST/ast.py

```diff
@@ -279,3 +280,23 @@ def visit(self) -> str:
         return (
             f'__classes__["{self._class_}"].members["{self.identifier}"] = ' + self.value.visit() + ";"
         )
+
+class MemberVarAssignNode(ASTNode):
+    def __init__(self):
+        super().__init__()
+        self.identifier = []
+        self._class_ = ''
+        self.value = ExprNode()
+        self.type = NodeTypes.VAR_ASSIGN
+
+    def visit(self) -> str:
+        result = ''
```

Comment:
in python there no need to declare string like this.

result += (here also no need to add string)